fe1-web/model/objects/wallet/Token.ts

```diff
@@ -1,12 +1,13 @@
 import { derivePath, getPublicKey } from 'ed25519-hd-key';
-import { EventStore, OpenedLaoStore, WalletStore } from 'store';
+import { OpenedLaoStore, WalletStore } from 'store';
+import { RollCall } from 'features/rollCall/objects';
```

Comment:
hopefully, at some point we will be able to get rid of the `RollCall` from here.